### ISSUE TITLE
xmlschema >=2.5.0 breaks mamba 1.4.2 in our test environment

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#2056]
recent mamba version is 1.5.1, 2023-10-03, currently not verified to be safe.

the old version mamba 1.4.2  gets broken by a build with an error related to an update of xmlschema

```
Extracting   (1) ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╸━━━━━━━━━━━━━       0 /       1 xmlschema                  0.0scritical libmamba Pathname can't be converted from UTF-8 to current locale.
error    libmamba Error opening for reading "/home/reimar/mambaforge/pkgs/xmlschema-2.5.0-pyhd8ed1ab_0/info/index.json": No such file or directory
xmlschema-2.5.0-pyhd8ed1ab_0.conda extraction failed
```

We have to update the  documentation regarding mambaforge is replaced by miniforge,
https://github.com/conda-forge/miniforge#faq

The interim solution is to pin xmlschema <2.5.0

seen first in https://github.com/Open-MSS/MSS/actions/runs/6440630490/job/17501580850?pr=2043

**Comment by ReimarBauer** [comment]
currently miniforge3 and mambaforge docker images have the 1.4.2 version of mamba

```
docker run --rm -it --entrypoint bash quay.io/condaforge/miniforge3
(base) root@af288fbcb9b0:/# mamba --version
mamba 1.4.2
conda 23.3.1
```

```
docker run --rm -it --entrypoint bash quay.io/condaforge/mambaforge
(base) root@fee352861e66:/# mamba --version
mamba 1.4.2
conda 23.3.1
```
so we can't update currently to miniforge3

**Comment by ReimarBauer** [comment]
https://github.com/conda-forge/mamba-feedstock/issues/193

**Comment by ReimarBauer** [comment]
docker  run --rm -it --entrypoint bash  condaforge/miniforge3:latest

current condaforge build is still based on 

mamba --version
mamba 1.4.2
conda 23.3.1


mamba 1.5.5 was recently released

**Comment by ReimarBauer** [comment]
this looks solved in the mamba 1.5.5. version, maybe also earlier. Can't reproduce in my local docker miniforge3 container.

**Comment by ReimarBauer** [comment]
unfortunately our docker mambaforge based container still has this xmlschema issue
